handler/src/test/java/io/netty/handler/ssl/OpenSslEngineTest.java

```diff
@@ -165,6 +166,13 @@ public void testSupportedSignatureAlgorithms(SSLEngineTestParam param) throws Ex
         super.testSupportedSignatureAlgorithms(param);
     }
 
+    @Override
+    protected void additionalPeerAssertions(SSLSession sslSession, boolean mutualAuth) {
+        if (sslSession instanceof OpenSslInternalSession) {
```

Comment:
this could still be `OpenSslSession` then I think.

Reply:
You are right... This is what happens if you refactor via the IDE ;)